[2022-09-04 00:00:03,481] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-09-03T00:00:00+00:00 [queued]>
[2022-09-04 00:00:03,491] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-09-03T00:00:00+00:00 [queued]>
[2022-09-04 00:00:03,491] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-04 00:00:03,492] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-09-04 00:00:03,492] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-04 00:00:03,509] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-09-03 00:00:00+00:00
[2022-09-04 00:00:03,515] {standard_task_runner.py:52} INFO - Started process 32161 to run task
[2022-09-04 00:00:03,517] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-09-03T00:00:00+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpo3riop8t', '--error-file', '/tmp/tmpci7sem_9']
[2022-09-04 00:00:03,519] {standard_task_runner.py:80} INFO - Job 319: Subtask link_dag
[2022-09-04 00:00:03,590] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-09-03T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-09-04 00:00:03,669] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-09-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-03T00:00:00+00:00
[2022-09-04 00:00:44,118] {python.py:173} INFO - Done. Returned value was: None
[2022-09-04 00:00:44,129] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220903T000000, start_date=20220904T000003, end_date=20220904T000044
[2022-09-04 00:00:44,172] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-04 00:00:44,199] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
